### PR TITLE
Add stack state file

### PR DESCRIPTION
**Pull request by nguyer**
This PR adds a stack file which contains information about the current state of the stack, including things that have been added after the stack has been started. Right now this includes:

- Accounts: address and private key (for Ethereum)
- Deployed custom contracts

This PR adds two new CLI commands:

 - `accounts create <stack_name>` - creates a new account, registers it in the appropriate manner for the current blockchain (Geth and Besu only right now), stores it in the stack state JSON, and prints it to stdout.
 - `accounts list <stack_name>` (also aliased as `accounts ls`) - lists all of the registered accounts in the given stack. Any one of these signing keys should be able to be used to submit a transaction.

### CODE REVIEW

File: internal/blockchain/ethereum/geth/geth_provider.go [hunk 1]
```diff
@@ -161,13 +161,13 @@ func (p *GethProvider) DeployFireFlyContract() (*core.BlockchainConfig, error) {
...
-			addresses = addresses + ","
-		}
-	}
+	// addresses := ""
+	// for i, member := range p.Stack.Members {
+	// 	addresses = addresses + member.Address
+	// 	if i+1 < len(p.Stack.Members) {
+	// 		addresses = addresses + ","
+	// 	}
+	// }
```
Comment (awrichar) [inline internal/blockchain/ethereum/geth/geth_provider.go:170]: Can this be deleted?

File: pkg/types/stack.go [hunk 2]
```diff
@@ -20,35 +20,37 @@ type Stack struct {
...
+	ExposedDataexchangePort   int         `json:"exposedDataexchangePort,omitempty"`
+	ExposedIPFSApiPort        int         `json:"exposedIPFSApiPort,omitempty"`
+	ExposedIPFSGWPort         int         `json:"exposedIPFSGWPort,omitempty"`
+	ExposedUIPort             int         `json:"exposedUiPort,omitempty"`
+	ExposedTokensPorts        []int       `json:"exposedTokensPorts,omitempty"`
+	External                  bool        `json:"external,omitempty"`
+	OrgName                   string      `json:"orgName,omitempty"`
+	NodeName                  string      `json:"nodeName,omitempty"`
+	// Address                   string `json:"address,omitempty"`
+	// PrivateKey                string `json:"privateKey,omitempty"`
```
Comment (awrichar) [inline pkg/types/stack.go:55]: Remove?

**Review by awrichar — APPROVED** [review]

File: internal/blockchain/ethereum/geth/geth_provider.go (hunk 1 above)
Comment (nguyer) [reply internal/blockchain/ethereum/geth/geth_provider.go:170]: Yes, thanks

File: pkg/types/stack.go (hunk 2 above)
Comment (nguyer) [reply pkg/types/stack.go:55]: Yep

**Pull request merged**